cli/command/context.go

```diff
@@ -37,8 +42,11 @@ func (dc *DockerContext) UnmarshalJSON(payload []byte) error {
 	if err := json.Unmarshal(payload, &data); err != nil {
 		return err
 	}
+	dc.Type = DefaultContextType
```

Comment:
Wondering if it would make sense to use `""` (empty string) as default. i.e., if no type is present, use the current default, and only require a value to be set for non-default. (We could still print "default" or "moby" when doing a `docker context ls` if we want)

Reply:
Yes, we can let "" (empty) be the default type especially for storing contexts, and only replace it when outputting context type to users, that's maybe closer to what already exists in the field for now

Reply:
Cool!
PTAL